### ISSUE TITLE
Spurious warnings about layout tokens in calc().

### ISSUE THREAD

**Issue by wkeese** [carbon-design-system/stylelint-plugin-carbon-tokens#126]
Given:

```scss
@use "@carbon/layout" as layout;
...
.foo {
  margin-top: calc(-1 * #{layout.$spacing-01});
}
```

We get the spurious carbon/layout-use error that:

>>   229:15  ⚠  Expected calc of the form calc(P O #{$}) or calc(-1 * #{$}). Where 'P' is in (vw, vh or %), 'O' is + or -,  '$' is a carbon layout token, mixin or function for "margin-top" found "calc(-1 * #{layout.$spacing-01})".

(In reality, `layout.$spacing-01` is a carbon layout token.)

**Comment by lee-chase** [comment]
The scopes functionality is somewhat broken, thanks for raising this issue.
